inspire/modules/migrator/manage.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Manage migration from INSPIRE legacy instance."""
+
+from __future__ import print_function
+
+import os
+import sys
+
+from flask import current_app
+from flask.ext.script import prompt_bool
+
+from invenio.ext.es import create_index as create_main_index
+from invenio.ext.es import delete_index as delete_main_index
+from invenio.ext.script import Manager
+from invenio.ext.sqlalchemy import db
+
+from invenio_records.api import Record
+
+from invenio_workflows.receivers import create_holdingpen_index
+from invenio_workflows.receivers import delete_holdingpen_index
+
+import six
+
+from werkzeug.utils import import_string
+
+from .tasks import migrate
+
+manager = Manager(description=__doc__)
+
+
+@manager.option('--records', '-r', dest='records',
+                action='append',
+                default=None,
+                help='Specific record IDs to migrate.')
+@manager.option('--collection', '-c', dest='collections',
+                action='append',
+                default=None,
+                help='Specific collections to migrate.')
+@manager.option('--input', '-f', dest='file_input',
+                help='Specific collections to migrate.')
+@manager.option('-t', '--input-type', dest='input_type', default='marcxml',
+                help="Format of input file.")
+def populate(records, collections, file_input=None, input_type=None):
+    """Train a set of records from the command line.
+
+    Usage: inveniomanage predicter train -r /path/to/json -o model.pickle
+    """
+    if records is None and collections is None:
+        # We harvest all
+        print("Migrating all records...", file=sys.stderr)
+    if records:
+        print("Migrating records: {0}".format(",".join(records)))
+    if collections:
+        print("Migrating collections: {0}".format(",".join(collections)))
+
+    if file_input and not os.path.isfile(file_input):
+        print("{0} is not a file!".format(file_input), file=sys.stderr)
+        return
+
+    if file_input:
+        print("Migrating records from file: {0}".format(file_input))
+        # FIXME: Add hook to allow for pre-allocation of IDs (--force)
```

Comment:
@kaplun @Panos512 Here you can add the signal hook discussed in #379. (perhaps by adding a `force` flag above). Use `action=store_true` for flags in the `manager.option`.